packages/react-native/ReactAndroid/cmake-utils/ReactNative-application.cmake

```diff
@@ -87,6 +87,11 @@ add_library(react_nativemodule_core ALIAS ReactAndroid::react_nativemodule_core)
 add_library(react_render_imagemanager ALIAS ReactAndroid::react_render_imagemanager)
 add_library(rrc_image ALIAS ReactAndroid::rrc_image)
 add_library(rrc_legacyviewmanagerinterop ALIAS ReactAndroid::rrc_legacyviewmanagerinterop)
+add_library(reactnativejni ALIAS ReactAndroid::reactnativejni)
```

Comment:
This feels wrong @WoLewicki, I don't believe you need it.
Are you sure about this target?

Reply:
It is needed in `TextLayoutManager` for `facebook::react::NativeArray::consume()`: https://github.com/facebook/react-native/blob/c4a6bbc8fd8992059abdcfae2bc483bd29f31e79/packages/react-native/ReactCommon/react/renderer/textlayoutmanager/platform/android/react/renderer/textlayoutmanager/TextLayoutManager.cpp#L193

![image](https://github.com/facebook/react-native/assets/32481228/2f4936b0-08d1-4f2e-a5d9-9254e5fcdb3b)